ext/mysql2/extconf.rb

```diff
@@ -125,7 +127,7 @@ def asplode(lib)
 end
 
 usable_flags = wishlist.select do |flag|
-  try_link('int main() {return 0;}', flag)
+  try_link('int main() {return 0;}', "-Werror -Wunknown-warning-option #{flag}")
```

Comment:
Yeah, that's true. It'd be good to surface that, but I think this is reasonable for now. It seems that there's some divergence between clang and gcc on this, in that gcc's version of this flag is or was `-Wunknown-warning` - I don't have gcc installed, so I can't check, but something to be aware of.